### DISCUSSION TITLE
SAVE button disabled on using Live or Post QSO after using TAB to go from callsign to another field on the same form

### DISCUSSION THREAD

**Discussion by PA4RB** [magicbug/Cloudlog#2018] [Q&A]
On using Live or Post QSO after using TAB or going with mouse from callsign to another field on the same form the SAVE button seems disabled. I can see the Button changing colours though but on clicking nothing happens. . When only putting in the callsign the SAVE button works.

On both Windows and Mac i experience this strange behaviour. Hope that someone can explain what might be the issue.

**Comment by magicbug** [comment]
Not quite sure I understand, save the will of course not be available if the callsign field is empty validation on the page will alert you to this

Other issues of QSO not saving is DXCC dropdown not being filled in but that doesn't sound like the problem in this case.

**Comment by PA4RB** [comment]
Hi Peter,

Thanks for quick reply.
Steps:
Fill callsign in callsign field
Use TAB or mouse to go to next field
Verify that QRZ or HamQTH info is gathered
Click Save
If all okay repeat steps
In my case the Save button then changes color on hovering but on clicking
nothing happens

When I fill in the callsign field and NOT using the mouse or TAB but going
directly to Save the QSO is saved

Regards Rob

Op di 28 feb. 2023 om 17:39 schreef Peter Goodhall ***@***.***

**Reply by PA4RB** [reply]
Just updated country files and rechecked all qso's in logbook. Everything seems alright again.
